tests/Doctrine/Tests/Common/Annotations/CachedReaderTest.php

```diff
@@ -95,34 +98,36 @@ public function testUsesFreshCacheWithTraitsThatUseOtherTraits()
 
     protected function doTestCacheStale($className, $lastCacheModification)
     {
-        $cacheKey = $className;
+        $cacheKey       = strtr($className, '\\', '.');
```

Comment:
why this extra indentation ?

Reply:
Probably a remnant from a previous version of the code, I will remove it.